charts/comet-site-v1/values.yaml

```diff
@@ -32,6 +32,15 @@ resources:
     cpu: 50m
     memory: 200Mi
 
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
```

Comment:
I doubt anybody would want to change name. We do not allow changing the name of any other resources.